DAO/project/contracts/DAO-contract/tests/utils/setup.rs

```diff
@@ -10,7 +10,7 @@ abigen!(
     ),
     Contract(
         name = "GovToken",
-        abi = "./contracts/DAO-contract/tests/artifacts/gov_token/out/debug/gov_token-abi.json"
+        abi = "./contracts/test-artifacts/gov_token/out/debug/gov_token-abi.json"
```

Comment:
Given the structure of the AMM it made sense to extract some code but here it does not. I would move the code back to where it was.

Iirc the token isn't even needed and can be replaced by the functionality from the SDK so I recommend either moving it back or deleting it and generating an asset to deposit via the SDK

Reply:
This is required due to changes with nested projects and forc workspaces. An error is produced if the test artifacts are kept in their old places.

Reply:
ugh, so you cannot have a project inside a project. they can only be siblings...
okay, two options, replace with the SDK or raise issues to address this when we have time.
thoughts?

Reply:
I've replaced with the SDK